### PR TITLE
Update database help message

### PR DESCRIPTION
**Pull request by xeruf**
Users are still pointed to some ancient mailing list no-one uses anymore.
I changed "contact" to "consult" rather than "see" because it is reasonably general and thus allows us to point to almost anything without having to change the wording again.

### CODE REVIEW

**Review by Holzhaus — APPROVED** [review]
Thanks. This generally LGTM, but I'm unsure if it makes sense to update the TS files as these are pull edfrom Transifex anyway IIRC.

**Comment by xerus2000** [comment]
Ah ok, I wasn't involved in the translation yet.

**Comment by uklotzde** [comment]
Those files are generated by Transifex. Modifying them locally is pointless.

**Comment by Holzhaus** [comment]
@xerus2000 Can you run `git reset HEAD~1 -- res/translations` to remove the translation changes then?

**Comment by xerus2000** [comment]
But how are the strings updated then? I don't see a need to retranslate it in all languages.

**Comment by uklotzde** [comment]
What's the reason for *2f5be78 Reorder schema update function signature*?

**Comment by xerus2000** [comment]
It is much more likely that you want to specify a different target schema version rather than a different schema file. Since you don't seem to like to have separate PRs for such things, I thought it would fit in here.

**Comment by uklotzde** [comment]
> It is much more likely that you want to specify a different target schema version rather than a different schema file. Since you don't seem to like to have separate PRs for such things, I thought it would fit in here.

This is just an assumption and guessing. Don't change existing code without any actual requirements.

File: src/database/mixxxdb.h [hunk 1]
```diff
@@ -19,8 +19,8 @@ class MixxxDb : public QObject {
 
     static bool initDatabaseSchema(
             const QSqlDatabase& database,
-            const QString& schemaFile = kDefaultSchemaFile,
-            int schemaVersion = kRequiredSchemaVersion);
+            int schemaVersion = kRequiredSchemaVersion,
```
Comment (uklotzde) [inline src/database/mixxxdb.h:22]: I do not agree to include unrelated and unnecessary changes in this PR.

**Review by uklotzde — CHANGES_REQUESTED** [review]

File: src/database/mixxxdb.h (hunk 1 above)
Comment (xerus2000) [reply src/database/mixxxdb.h:22]: > This is just an assumption and guessing.

Nope, I actually wanted to change the target version in a test - but I didn't want to change the file, so I would have had to specify the path again.
Here's the thing: If you supply a different schema file, versions don't mean the same thing, so using the default required version wouldn't make sense. On the other hand, targeting a different version in the same file is a totally valid scenario.

Comment (uklotzde) [reply src/database/mixxxdb.h:22]: Ok. This information was missing.

File: src/track/trackid.h [hunk 2]
```diff
@@ -1,10 +1,7 @@
-#ifndef TRACKID_H
```
Comment (uklotzde) [inline src/track/trackid.h:1]: Please avoid to include unrelated changes in future PRs.

**Review by uklotzde — APPROVED** [review]

**Comment by uklotzde** [comment]
LGTM
